provider/terraform/website/docs/r/compute_instance_group_manager.html.markdown

```diff
@@ -90,14 +90,15 @@ The following arguments are supported:
     name.
 
 * `instance_template` - (Optional) The full URL to an instance template from
-    which all new instances will be created. Conflicts with `version` (see [documentation](https://cloud.google.com/compute/docs/instance-groups/updating-managed-instance-groups#relationship_between_instancetemplate_properties_for_a_managed_instance_group))
+    which all new instances will be created. This field is only present in the
+    `google` provider.
 
 * `version` - (Optional) Application versions managed by this instance group. Each
     version deals with a specific instance template, allowing canary release scenarios.
-    Conflicts with `instance_template`. Structure is documented below. Beware that
-    exactly one version must not specify a target size. It means that versions with
-    a target size will respect the setting, and the one without target size will
-    be applied to all remaining Instances (top level target_size - each version target_size).
+    Structure is documented below. Beware that exactly one version must not specify a
```

Comment:
While you're here, any thoughts on a better way to word this? It's fairly hard to understand as-is (no worries if not, I know you didn't write it)

Reply:
Sure, I took a crack at it - let me know your thoughts.

Reply:
That's a lot better, thanks!